doc/Developing.md

```diff
@@ -116,16 +125,17 @@ To build and run the browser example:
 
     git clone https://github.com/theia-ide/theia \
     && cd theia \
-    && npm install --unsafe-perm\
-    && cd ../../examples/browser \
+    && npm install --unsafe-perm \
```

Comment:
BTW we discussed this a bit and --unsafe-perm looks really weird.
I Suggest we move it out of the TL;DR and add it to the troubleshooting section
but that can be in another PR